picard/acoustid/__init__.py

```diff
@@ -108,42 +106,16 @@ def _on_lookup_finished(self, task, document, http, error):
                 mparms,
                 echo=None
             )
+            task.next_func(doc, http, error)
```

Comment:
`doc` is always `{}`, expected?

Reply:
Yes, it's in the error case empty. It was the same before. But it also was confusing (and partially used wrong) with two similarly named variables `document` and `doc` with different purpose. I cleaned this up and removed the `doc` variable.